yb-voyager/src/utils/utils.go

```diff
@@ -337,6 +337,51 @@ func ContainsAnySubstringFromSlice(slice []string, s string) bool {
 	return false
 }
 
+func WaitForLineInLogFile(filePath string, message string, timeoutDuration time.Duration) error {
+	// Wait for log file to be created
+	timeout := time.After(timeoutDuration)
+	for {
+		_, err := os.Stat(filePath)
+		if err == nil {
+			break
+		}
+		select {
+		case <-timeout:
+			return fmt.Errorf("timeout while waiting for log file %q", filePath)
+		default:
+			time.Sleep(1 * time.Second)
+		}
+	}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return fmt.Errorf("error opening file %s: %v", filePath, err)
+	}
+
+	defer file.Close()
+
+	for {
```

Comment:
I have a question here:
`bufio.Scan()` will return false in EOF is reached. Have we considered the case where we read the log file faster than the rate of generation of log lines. Because that will cause this to exit(or error out) before even reaching the required point.